### PR TITLE
Checkin STARCORE project OE52_OE54_ACVP effort

### PR DESCRIPTION
**Pull request by kaleb-himes**
# Description

CodeWarrior project files for OE52_OE54_ACVP effort on STARCORE SC3850 DSP

FIPS validation testing for OE addition

# Checklist

 - [X] added tests
 - [ ] updated/added doxygen
 - [X] updated appropriate READMEs
 - [X] Updated manual and documentation (FIPS UG and SP)

### CODE REVIEW

File: IDE/STARCORE/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.metadata/.plugins/org.eclipse.core.resources/.history
```
Comment (dgarske) [inline IDE/STARCORE/.gitignore:1]: Is the `.metadata` folder really required?

File: IDE/STARCORE/testwolfcrypt_8254/LCF/mmu_attr.l3k [hunk 2]
```diff
@@ -0,0 +1,128 @@
+// mmu_attr_cpp.l3k
+// *** Copyright Freescale Semiconductor= Inc= 2010 ***
```
Comment (dgarske) [inline IDE/STARCORE/testwolfcrypt_8254/LCF/mmu_attr.l3k:2]: I don't think we can handle this copyright.

**Review by dgarske — CHANGES_REQUESTED** [review]

File: IDE/STARCORE/.gitignore (hunk 1 above)
Comment (kaleb-himes) [reply IDE/STARCORE/.gitignore:1]: Yes, the metadata contains all the rules for locating the source and headers along with project configuration settings etc.

File: IDE/STARCORE/testwolfcrypt_8254/LCF/mmu_attr.l3k (hunk 2 above)
Comment (kaleb-himes) [reply IDE/STARCORE/testwolfcrypt_8254/LCF/mmu_attr.l3k]: Copyright files removed.

**Review by dgarske — CHANGES_REQUESTED** [review]
Please avoid adding .7z to repo. Thanks

**Comment by kaleb-himes** [comment]
TODO:

- Update FIPS user guide removing instructions regarding .7z  locations in IDE/
- Place .7z archives in FIPS repo
- Modify FIPS bundle procedure for releases, unarchive from FIPS repo, place in IDE/STARCORE during release bundling

**Pull request closed without merging**